examples/platforms/da15000/README.md

```diff
@@ -34,45 +6,48 @@ This directory contains example platform drivers for the [Dialog Semiconductor D
 
 ## Build Examples (How to build and flash):
 
-# cd <path-to-openthread>
+```
+$ cd <path-to-openthread>
 $ ./bootstrap
 $ make -f examples/Makefile-da15000 clean
 $ make -f examples/Makefile-da15000
+```
 
 Execute script to prepare image for flashing:
 
+```
 $ cd ./third_party/dialog/DialogTools
 $ ./imgprep.sh
+```
 
 Flash Binaries:
 
 1. Open another terminal window in order to start gdb server. Write the command (Linux):
+```
 $ JLinkGDBServer -ir -if swd -speed auto -device Cortex-M0 -logtofile on -localhostonly
+```
 
 2. Return to previous terminal window and execute a command in order to flash the board:
```

Comment:
This starts a new list, back at 1 again, since it is not continuous with the list above.